pkg/db/common/tasks.go

```diff
@@ -50,7 +50,7 @@ func CreateTaskFilter(pageIdx int, pageSize int, sort string, sortOrder string)
 		f.PageIndex = 0
 	}
 
-	if f.PageSize <= 0 || f.PageSize >= 50 {
+	if f.PageSize <= 0 || f.PageSize > 50 {
```

Comment:
Can you handle a config param (in the web section) about this max value?
Easier to tune.